.github/workflows/production-build.yaml

```diff
@@ -1,11 +1,20 @@
 name: Unstract Docker Image Build and Push (Production)
 
 on:
+  workflow_dispatch:
+    inputs:
+      tag:
+        description: "Docker image tag"
+        required: true
+      branch:
+        description: "Branch to be used"
+        required: true
+
   release:
     types:
       - created
 
-run-name: "[${{ github.event.release.tag_name }}] Docker Image Build and Push (Production)"
+run-name: "[${{ github.event.release.tag_name || github.event.inputs.branch }}] Docker Image Build and Push (Production)"
```

Comment:
@jaseemjaskp @johnyrahul we should be using tag only. Branch shouldn't be an input

Reply:
done.